### ISSUE TITLE
Q: Is it normal behavior to not show an event in the Day view under the following conditions...

### ISSUE THREAD

**Issue by HeyWrecker** [Serhioromano/bootstrap-calendar#664]
I'm making use of your Bootstrap Calendar plugin (it's awesome, thanks!) and am running into a display issue. We're allowing individuals to schedule events up to 8 hours long from just about any start time. To give a specific example:

Start Time: 3:30 PM (15:30)
Duration: 8 hours
Calculated End Time: 11:30 (23:30)

The event does not show up in the Month or Day view, however it does show up in the Week view. To the best of my knowledge, we're running with a default install.

Is this normal behavior? Why would an event like that show up on a day in the Weekly view, but not in the Monthly or Daily views?

Thanks!

**Comment by Serhioromano** [comment]
No, this is not normal. Perhaps some bug. I was accepting few PRs recently without actually testing everything myself. I can assume that there is very hi chance there is some bug.
